aat/src/aat/resources/features/F-083/F-083.feature

```diff
@@ -0,0 +1,43 @@
+@F-083
+Feature: Fetch the UI definition for the search inputs for a given Case Type :: GET /api/display/search-input-definition/{id}
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-359
+  Scenario: must return the UI definition of search inputs for a given case type
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains a valid case type id]
+    And it is submitted to call the [fetch UI definition for search inputs] operation of [CCD Definition Store]
+    And the response [has the 200 OK code]
+    And the response has all other details as expected
+
+  @S-356 @Ignore @RDM-7618
+  Scenario: must return 401 when request does not provide valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains an invalid authentication credentials]
+    And it is submitted to call the [fetch UI definition for search inputs] operation of [CCD Definition Store]
+    Then a negative response is received
+    And the response [contains 401 unauthorised code]
+    And the response has all the details as expected
+
+  @S-357 @Ignore @RDM-7618
+  Scenario: must return 403 when request provides authentic credentials without authorised access to the operation 
```

Comment:
I don't think it's a valid scenario for this endpoint. In definition-store only `/import` and admin-web endpoints (/api/draft) are protected with specific user roles - rest can be accessible by any idam user .